app/src/main/res/layout/activity_dashboard.xml

```diff
@@ -9,7 +9,7 @@
         <variable name="selectedPage" type="LiveData&lt;org.cru.godtools.ui.dashboard.Page&gt;" />
     </data>
 
-    <androidx.drawerlayout.widget.DrawerLayout
+    <org.ccci.gto.android.common.androidx.drawerlayout.widget.HackyDrawerLayout
```

Comment:
Could I ask what makes this a hacky implementation or should I just go take a look at it?

Reply:
it's an android bug with multi-touch motion events, sometimes invalid events are fed into the touch methods and it causes the app to crash. We are basically hacking around the framework bug by looking for those crashes and suppressing them so the app will continue to function.

Reply:
https://github.com/CruGlobal/android-gto-support/blob/master/gto-support-androidx-drawerlayout/src/main/java/org/ccci/gto/android/common/androidx/drawerlayout/widget/HackyDrawerLayout.kt#L16-L21